lib/fs/casefs.go

```diff
@@ -375,7 +376,10 @@ func (f *caseFilesystem) checkCaseExisting(name string) error {
 	if err != nil {
 		return err
 	}
-	if realName != name {
+	// We normalize the normalization (hah!) of the strings before
```

Comment:
Obligatory xkcd: https://xkcd.com/927/  
(And new standard: standard == normalisation (== normalization)).